src/Controls/src/Core/Handlers/Items/Android/SizedItemContentView.cs

```diff
@@ -29,10 +29,10 @@ protected override void OnMeasure(int widthMeasureSpec, int heightMeasureSpec)
 			double targetHeight = _height();
 
 			if (!double.IsInfinity(targetWidth))
-				targetWidth = (int)Context.FromPixels(targetWidth);
+				targetWidth = Context.FromPixels(targetWidth);
```

Comment:
I think based on this changes, there are three failing tests becuase small differences:
<img width="339" alt="image" src="https://github.com/user-attachments/assets/034c8ba9-0fb2-40a2-80b3-9842ac592faa" />

For example:
CarouselItemsShouldRenderProperly
`Snapshot different than baseline: CarouselItemsShouldRenderProperly.png (1.13% difference)`
![image](https://github.com/user-attachments/assets/de6bc891-c03c-461c-b47a-7db743df7f2f)

(In red the differences)

Could you review/update the snapshots? Let me know if can help.

Reply:
I updated the screenshots.